game/templates/game/rules.html

```
@@ -1,10 +1,12 @@
+{% load static %}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)